transport/tchannel/handler.go

```diff
@@ -98,7 +98,8 @@ func (h handler) handle(ctx context.Context, call inboundCall) {
 	}
 
 	if _, ok := err.(tchannel.SystemError); ok {
-		call.Response().SendSystemError(err)
+		// TODO: explicitly ignoring error, can we deal with this?
+		_ = call.Response().SendSystemError(err)
```

Comment:
We can't deal with this because the only way to handle this is let the handler
crash and make the TChannel server try to send an error frame--which will fail
because that's exactly what this line is doing.